src/ol/index.modern.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
Did you mean to include this file?

Reply:
That was an accident. Thanks for your sharp eye! I force-pushed a version without that file.